test/integration/user_can_see_if_item_is_retired_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class UserCanSeeIfItemIsRetiredTest < ActionDispatch::IntegrationTest
+  test 'user can see retired item status on order show page' do
+    create_shop_and_logged_in_user
+
+    within(".old-chips") do
+        assert page.has_content?("retired")
+    end
+  end
+
+  test 'user can see retired item status on chip show pages' do
+    create_shop_and_logged_in_user
+
+    visit '/chips'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.